### DISCUSSION TITLE
Can We Implement in .Net Maui like "NativeView.ToMauiPlaform();

### DISCUSSION THREAD

**Discussion by nextcodelab** [dotnet/maui#5989] [Q&A] [answered]
In Color you can covert to plaform like this.. nativeColor.ToPlatform()

It is posible to program in C# that the Native UI can be converted on Maui View like just an extension or other technique that produce like this..

~~~cs
#if Android
        NavigationView nativeUIIntance = new NavigationView (Context);
        var mauiView = nativeUIIntance.ToMauiView();
#elif Windows
       NavigationView nativeUIIntance = new NavigationView ();
       var mauiView = nativeUIIntance.ToMauiView();
#endif
~~~

**Comment by pictos** [comment, marked as answer]
Are you looking for this?
https://github.com/dotnet/maui/pull/5218#issuecomment-1064712417
